src/components/common/Extensions/ExtensionActionButton/ExtensionEnableButton.tsx

```diff
@@ -26,6 +39,20 @@ const EnableButton = ({ extensionData, inputDisabled, stopPolling }: Props) => {
     );
   };
 
+  // If extension is not initializable but is missing permissions, show enable extension action button
+  if (
+    !extensionData.initializationParams &&
+    extensionData.missingColonyPermissions.length
+  ) {
+    return (
+      <ActionButton
+        text={MSG.enable}
+        actionType={ActionTypes.EXTENSION_ENABLE}
+        values={{ colonyAddress: colony.colonyAddress, extensionData }}
+      />
+    );
+  }
+
```

Comment:
The lack of this was preventing us from setting permissions for non-initializable extensions, such as Staged Expenditure